### PR TITLE
jsapi-4.4

### PR DESCRIPTION
**Pull request by jwasilgeo**
After this is accepted and merged, I will do the following (likely not until week of 7/17):

- prep the repo for a new v2.x release (package.json)
- publish a new v2.x release to github and npm
- update the this repo's gh-pages branch so that the public docs site has all the latest and greatest

### CODE REVIEW

File: src/core/esriLoader.js [hunk 1]
```diff
@@ -22,7 +22,7 @@
          * Loads the ArcGIS API for JavaScript.
          *
          * @param {Object=} options Send a list of options of how to load the ArcGIS API for JavaScript.
-         *  This defaults to `{url: '//js.arcgis.com/4.3'}`.
```
Comment (andygup) [inline src/core/esriLoader.js]: Consider adding the protocol here as well `https`

File: src/core/esriLoader.js [hunk 2]
```diff
@@ -22,7 +22,7 @@
...
+         *  This defaults to `{url: '//js.arcgis.com/4.4'}`.
```
Comment (andygup) [inline src/core/esriLoader.js]: Ditto on adding protocol `https`

**Comment by jwasilgeo** [comment]
Thank you, @andygup. How about now?

**Comment by andygup** [comment]
Looks good!

**Pull request merged**